Assignment/starter_code/portfolio.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta lang="en">
+	<title>Esther Butcher -- Portfolio</title>
+	<link href="./style.css" rel="stylesheet">
+</head>
+<body>
+	<header>
+		<nav>
+			<ul class="links">
+				<li><a href="index.html">About</a></li>
+				<li>Experience</li>
+			</ul>
+		</nav>
+		<h2>Esther Butcher</h2>
+	</header>
+	<article>
+		<h1>Experience</h1>
+		<p>I recently graduated from a Front-End Web Development course at General Assembly wehre I learned HTML, CSS, JavaScript, jQuery and learned hot to be a <strong>fully sick rockstar ninja</strong> Front-End Web Developer. During my spare time I ride ponies and do colouring in.</p>
+		
+		<h3>Relevant Experience</h3>
+		<ul>
+			<li>
+				<p><strong>Citipix</strong>, Web Developer, Relaxr 2015 &ndash; present<br />
+				Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</p>
+				<img src="images/citipix_deliverable.png" alt="citipix screenshot">
+			</li>
+			<li>
+				<p><strong>Relaxr</strong>, Web Developer, Startup Matchmaker 2015 &ndash; present<br />
+				Used a design team's wireframes to develop this company's responsive homepage.</p>
+				<img src="images/relaxr_deliverable.png" alt="Relaxr screenshot">
+			</li>
+			<li>	
+				<p><strong>Startup Matchmaker</strong>, Web Developer, CitiPix 2015 &ndash; present<br />
```

Comment:
You should be able to use ```-``` instead of ```&ndash;```

The only times we use the ```&xxx;``` is for 

```
&lt; (<)
&gt; (>)
&amp; (&)
```

because they are read the by the browser as html syntax